azure-pipelines.yml

```diff
@@ -204,38 +204,34 @@ stages:
         -TsaCodebaseName "Arcade-Validation"
         -TsaPublish $True'
 
-  - stage: Push_to_latest_channel
-    displayName: Push build to 'Latest' channel
-    dependsOn:
-      # This will run only after all the publishing stages have run.
-      # These stages are introduced in the eng/common/templates/post-build/channels YAML templates
-      - NetCore_Dev5_Publish
-      - Net5_Preview1_Publish
-      - Net5_Preview2_Publish
-      - Net_Eng_Latest_Publish
-      - Net_Eng_Validation_Publish
-      - General_Testing_Publish
-      - NETCore_Tooling_Dev_Publishing
-      - NETCore_Tooling_Release_Publishing
-      - NET_Internal_Tooling_Publishing
-      - NETCore_Experimental_Publishing
-      - Net_Eng_Services_Int_Publish
-      - Net_Eng_Services_Prod_Publish
-    jobs:
+# Arcade validiation with additional repos
+- stage: Additional_Arcade_Validation
```

Comment:
This probably needs a more descriptive stage and displaynames. having a stage called addition validation in a build that already has a validate stage in a repo called arcade-validation. It's like the word has lost all meaning.

Something like validate_consumer_repos?

Reply:
![image](https://user-images.githubusercontent.com/47990216/77183994-5f983e00-6a8c-11ea-8f6c-6981489dc3a7.png)

Reply:
Renamed to "Validate Arcade with Consumer Repositories"